examples/react-alt/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+    "alt": "^0.17.3",
+    "director": "^1.2.0",
+    "react": "^0.13.3",
```

Comment:
0.14 doesn't ship with JSXTransformer, so that's why I didn't upgrade. I tried using Babel's in-browser transformer but didn't get it to immediately work so I gave it up for now.